### ISSUE TITLE
Navigation of virtual keyboard with arrow keys doesn't work

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetML#747]
One can navigate the virtual keyboard with the Tab key, except that one cannot change keyboard layouts, and it is really a pain to tab through all the key selections. Instead, one should be able to use the arrow keys to move among the keyboard options.

The arrows keys do work to change the keyboard layouts (unlike tab). Although the styling of the layout headings does change as they layouts change, the "selected" frame does not move to the right heading (unless one tabs to the keyboard itself and then shift-tabs back).
